README.md

```diff
@@ -138,6 +138,19 @@ slackSend(channel: slackResponse.channelId, message: "Update message now", times
 
 This feature requires [botUser](#bot-user-mode) mode.
 
+#### Emoji Reactions
+
+Add an emoji reaction to a previously-sent message like this:
+
+Example:
+
+```groovy
+def slackResponse = slackSend(channel: "emoji-demo", message: "Here is the primary message")
+slackResponse.addReaction("thumbsup")
+```
+
+This may only work reliably in channels (as opposed to private messages) due to [limitations in the Slack API](https://api.slack.com/methods/chat.postMessage) (See "Post to an IM channel").
+
```

Comment:
There are comments elsewhere in the readme about requiring bot user mode. I'm not actually sure whether this feature requires it. How can I find out? The [API docs](https://api.slack.com/methods/reactions.add) say it works for bot or user tokens.

Reply:
This needs to mention the scope needed for `reactions:write`